zuul.d/base.yaml

```diff
@@ -305,4 +307,4 @@
     pre-run:
       - ci/playbooks/e2e-prepare.yml
       - ci/playbooks/dump_zuul_data.yml
-      - ci/playbooks/pre-ci-bootstrap.yml
+      - ci/playbooks/bootstrap-networking-mapper.yml
```

Comment:
came here via slack request, so i don't have the context. lgtm (would be nice to have more of a commit message for explanation) and i cannot find any other references to pre-ci-bootstrap than this one with code search https://github.com/search?q=org%3Aopenstack-k8s-operators%20pre-ci-bootstrap.yml&type=code

Reply:
@marios thanks for having a look. I've added more information in the commit message.
Yes, that playbook and the job itself, weren't used, so we can now tweak everything without worrying about breaking working jobs.

Reply:
@marios could you please leave a /lgtm comment :)